data/seeds/virtualChild.js

```diff
@@ -0,0 +1,13 @@
+
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('VirtualChild').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('VirtualChild').insert([
+        {id: 1, email: '', first_name:"Mike",last_name:"McVirtual"},
+        {id: 2, email: '', first_name:"Sara",last_name:"McVirtual"},
+        {id: 3, email: '', first_name:"Jamie",last_name:"McVirtual"}
```

Comment:
Let's replace the contents of this file with what you added in the children seed file, then revert the children seed file back to how it was before. Also, if you can structure this file like how the parents seed file was structured, I think that would be great.